### PR TITLE
Feature/profile ui changes

### PR DESCRIPTION
**Pull request by defraimo**
- Address in profile Asset replaced
- Changed Asset and added a function in Images.swift for profile background image
- In Profile: changed view colour to white + changed size of profile image
-In Profile: changed label fonts and colours

### CODE REVIEW

File: Dizzy/Screens/Login/LoginVC.swift [hunk 1]
```diff
@@ -153,7 +153,7 @@ final class LoginVC: UIViewController, LoadingContainer, PopupPresenter {
     }
     
     private func setupNavigationView() {
-        self.navigationItem.title = "Login".localized
+        self.navigationItem.title = "".localized
```
Comment (menesh2) [inline Dizzy/Screens/Login/LoginVC.swift]: please remove this line, if we dont want any title we should not set it

File: Dizzy/Screens/SignUp/SignUpWithDizzyVC.swift [hunk 2]
```diff
@@ -50,7 +50,7 @@ final class SignUpWithDizzyVC: UIViewController, KeyboardDismissing, LoadingCont
     }
     
     private func setupNavigationView() {
-        self.navigationItem.title = "Sign Up".localized
+        self.navigationItem.title = "".localized
```
Comment (menesh2) [inline Dizzy/Screens/SignUp/SignUpWithDizzyVC.swift]: same here, if we dont want title we should not set it

**Review by menesh2 — CHANGES_REQUESTED** [review]
nice work! please read my comments

File: Dizzy/Screens/Login/LoginVC.swift (hunk 1 above)
Comment (Benlevy1025) [reply Dizzy/Screens/Login/LoginVC.swift]: done

File: Dizzy/Screens/SignUp/SignUpWithDizzyVC.swift (hunk 2 above)
Comment (Benlevy1025) [reply Dizzy/Screens/SignUp/SignUpWithDizzyVC.swift]: done

**Review by menesh2 — APPROVED** [review]
nice fixes!!

**Pull request merged**